snapshots/smoking/2024-05-30/cigarette_sales.zip.dvc

```diff
@@ -0,0 +1,28 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: International Smoking Statistics
+    date_published: "2016-12-19"
+    title_snapshot: International Smoking Statistics - Cigarette Sales
+
+    # Citation
+    producer: Barbara Forey, Jan Hamling, John Hamling, Alison Thornton, Peter Lee
```

Comment:
The field `producer` should be the name of the institution. If it was a paper, it would be something like `Forey et al.`.
In this case, it's neither an institution nor a paper. So I think it should be "P N Lee Statistics and Computing Ltd" (which is not a great name, in my opinion, but that's what their main page seems to show!). Alternatively, `Forey et al.` would be a fine choice (I leave it up to you).